packages/frames.js/src/render/types.ts

```diff
@@ -1,6 +1,10 @@
 import type { Frame, FrameButton, TransactionTargetResponse } from "..";
 import { FarcasterFrameContext } from "./farcaster";
 
+export type onTransactionFunc = (
```

Comment:
maybe a nit picking but in order to be consistent I think it would be better to have this Capitalized